egov-indexer/fire-noc-service.yml

```diff
@@ -0,0 +1,157 @@
+            uriResponseMapping:
+            - inJsonPath: $.ProcessInstances
+              outJsonPath: $.Data.history
+     
+
+  - topic: update-fn-firenoc
+    configKey: INDEX
+    indexes:
+      - name: firenoc-services
+        type: general
+        id: $.id,$.tenantId
+        jsonPath: $.FireNOCs
+        timeStampField: $.auditDetails.lastModifiedTime
+        customJsonMapping:
+          indexMapping: {"Data":{"workflow": {"state": {}, "action": "", "assignes": []},"id":"","tenantId":"","provisionFireNOCNumber":"","fireNOCDetails":{"status":"","action":"","fireNOCType":"","buildings":[],"usageType":"","propertyDetails":{},"additionalDetail":{"assignee":[]}},"applicantDetails":{"owners":[{"type":""}],"additionalDetail":{}},"auditDetails":{},"auditDetails":{"createdTime":""},"@timestamp":"timestamp"}}
+          fieldMapping:
+          - inJsonPath: $.fireNOCDetails.status
+            outJsonPath: $.Data.workflow.state
+          - inJsonPath: $.fireNOCDetails.action
+            outJsonPath: $.Data.workflow.action
+          - inJsonPath: $.fireNOCDetails.additionalDetail.assignes
+            outJsonPath: $.Data.workflow.assignes
+          - inJsonPath: $.id
+            outJsonPath: $.Data.id
+          - inJsonPath: $.provisionFireNOCNumber
+            outJsonPath: $.Data.provisionFireNOCNumber
+          - inJsonPath: $.tenantId
+            outJsonPath: $.Data.tenantId
+          - inJsonPath: $.fireNOCDetails.status
+            outJsonPath: $.Data.fireNOCDetails.status
+          - inJsonPath: $.fireNOCDetails.fireNOCType
+            outJsonPath: $.Data.fireNOCDetails.fireNOCType
+          - inJsonPath: $.fireNOCDetails.buildings
+            outJsonPath: $.Data.fireNOCDetails.buildings
+          - inJsonPath: $.fireNOCDetails.usageType
+            outJsonPath: $.Data.fireNOCDetails.usageType
+          - inJsonPath: $.fireNOCDetails.propertyDetails
+            outJsonPath: $.Data.fireNOCDetails.propertyDetails
+          - inJsonPath: $.applicantDetails.owners.type
+            outJsonPath: $.Data.applicantDetails.owners.type
+          - inJsonPath: $.applicantDetails.additionalDetail
+            outJsonPath: $.Data.applicantDetails.additionalDetail
+          - inJsonPath: $.applicantDetails.auditDetails
+            outJsonPath: $.Data.applicantDetails.auditDetails
+          - inJsonPath: $.auditDetails.createdTime
+            outJsonPath: $.Data.auditDetails.createdTime  
+          - inJsonPath: $.@timestamp
+            outJsonPath: $.Data.@timestamp
+          externalUriMapping:
+          - path: http://egov-workflow-v2.egov:8080/egov-workflow-v2/egov-wf/process/_search
+            queryParam: businessIds=$.applicationNo,history=true,tenantId=$.tenantId
```

Comment:
check whether the applicationNo filed is present in the fire noc json body or some other field